scalanativelib/src/ScalaNativeModule.scala

```diff
@@ -166,7 +166,8 @@ trait TestScalaNativeModule extends ScalaNativeModule with TestModule { testOute
     val testClassloader =
     new URLClassLoader(runClasspath().map(_.path.toIO.toURI.toURL).toArray,
       this.getClass.getClassLoader)
-    val frameworkInstances = TestRunner.frameworks(testFrameworks())(testClassloader)
+    val 
+     = TestRunner.frameworks(testFrameworks())(testClassloader)
```

Comment:
Is a val name missing here?

Reply:
I deleted some text by mistake apparently 🙈